includes/autoloader.php

```diff
@@ -154,18 +154,17 @@ private static function init_classes_map() {
 	/**
 	 * Normalize Class Name
 	 *
-	 * Used to convert control names to class name,
-	 * a ucwords polyfill for php versions not supporting delimiter parameter
-	 * reference : https://github.com/elementor/elementor/issues/7310#issuecomment-469593385
+	 * Used to convert control names to class names.
+	 *
+	 * @since 3.1.0
```

Comment:
This isn't the correct `@since`

Reply:
Removed. There wasn't a `@since` tag beforehand.